src/backend/prisma/schema.prisma

```diff
@@ -10,29 +7,172 @@ generator client {
-  updatedAt DateTime @updatedAt
-  title     String
-  content   String?
-  published Boolean  @default(false)
-  author    User     @relation(fields: [authorId], references: [id])
-  authorId  Int
+enum CR_Type {
+  DESIGN_ISSUE
+  NEW_FUNCTION
+  OTHER
+  STAGE_GATE
+  WP_ACTIVATION
+}
+
+enum WBS_El_Status {
+  INACTIVE
+  ACTIVE
+  COMPLETE
 }
 
-model Profile {
-  id     Int     @id @default(autoincrement())
-  bio    String?
-  user   User    @relation(fields: [userId], references: [id])
-  userId Int     @unique
+enum Role {
+  ADMIN
+  LEADERSHIP
+  PROJECT_MANAGER
+  PROJECT_LEAD
+  MEMBER
+  GUEST
 }
 
 model User {
-  id      Int      @id @default(autoincrement())
-  email   String   @unique
-  name    String?
-  age     Int?
-  posts   Post[]
-  profile Profile?
+  user_id           Int                @id @default(autoincrement())
+  first_name        String
+  last_name         String
+  email_id          String             @unique
+  last_login        DateTime
+  role              Role
+  Fab_Weld_Requests Fab_Weld_Request[]
+  Change_Requests   Change_Request[]
+  Stage_CR_Reviews  Stage_Gating[]
+  WP_Init_Positions WP_Initiation[]
+  Change            Change[]
+  WBS_Element       WBS_Element[]
+  Project           Project[]
+  WP_Initiation     WP_Initiation[]    @relation("project_lead")
+}
+
+model Fab_Weld_Request {
+  fw_id            Int         @id @default(autoincrement())
+  date_submitted   DateTime    @default(now())
+  part_name        String
+  quantity         Int
+  resources_needed String
+  link             String
+  date_completed   DateTime
+  requestor        User        @relation(fields: [requestor_id], references: [user_id])
+  requestor_id     Int
+  wbs_el           WBS_Element @relation(fields: [wbs_el_id], references: [wbs_el_id])
+  wbs_el_id        Int
+}
+
+model Change_Request {
+  cr_id          Int             @id @default(autoincrement())
+  date_submitted DateTime        @default(now())
+  type           CR_Type
+  date_reviewed  DateTime
+  accepted       Boolean?
+  review_notes   String?
+  changes        Change[]
+  requestor      User            @relation(fields: [requestor_id], references: [user_id])
+  requestor_id   Int
+  wbs_el         WBS_Element     @relation(fields: [wbs_el_id], references: [wbs_el_id])
+  wbs_el_id      Int
+  Scope_CR       Scope_CR[]
+  stage_gating   Stage_Gating[]
+  wp_init        WP_Initiation[]
+}
+
+model Scope_CR {
+  scope_cr_id     Int            @id
+  what            String
+  why             String
+  timeline_impact Int
+  budget_impact   Int
+  scope_impact    String
+  doc             String
+  change_request  Change_Request @relation(fields: [cr_id], references: [cr_id])
+  cr_id           Int
+}
+
+model Stage_Gating {
+  stage_gating_id    Int            @id
+  done               Boolean
+  leftover_budget    Int
+  design_reviewer    User           @relation(fields: [design_reviewer_id], references: [user_id])
```

Comment:
I don't really think that is possible. The syntax above is for Prisma to know that this table will have a foreign key that is of type User, and in general, I don't think it's possible to have an array of foreign keys. I guess because this is a many to many relationship it might require having to make a Design Reviewer table mapping user ids to stage_gating_ids.

Reply:
Addressed by commit b386f59a908c023f8913ae7459f3e18155d71f58